jdaviz/configs/default/plugins/export/export.py

```diff
@@ -231,6 +231,9 @@ def export(self, filename=None, show_dialog=None):
         filename : str, optional
             If not provided, plugin value will be used.
         """
+        if self.viewer_invalid_msg != "":
+            raise ValueError(self.viewer_invalid_msg)
+

```

Comment:
the rest are within their respective if-blocks (although this should be emptied if no viewer is selected, so its probably effectively the same... just might be easier to read for consistency when we eventually add multiselect support)